toketmaster/tests/test_signing.py

```diff
@@ -0,0 +1,36 @@
+import unittest
```

Comment:
Do we need `unittest`?

Reply:
Yeah, it organizes the test cases much nicer